presto-main/src/main/java/com/facebook/presto/execution/QueryStateMachine.java

```diff
@@ -693,4 +723,14 @@ public void pruneQueryInfo()
         );
         finalQueryInfo.compareAndSet(finalInfo, Optional.of(prunedQueryInfo));
     }
+
+    private long tickerNanos()
```

Comment:
This method is simple enough that I don't see much value in making it a function. I would inline it. Let me know if you see particular benefits in abstracting it here.